dspace/config/dspace.cfg

```diff
@@ -28,8 +28,8 @@ dspace.hostname = localhost
 # DSpace base host URL.  Include port number etc.
 dspace.baseUrl = http://localhost:8080
 
-# The user interface you will be using for DSpace. Common usage is either xmlui or jspui
-dspace.ui = xmlui
+# The user interface you will be using for DSpace.
+dspace.ui = dspace-spring-rest
```

Comment:
Again, as noted in #1889, "dspace-spring-rest" isn't a "ui".  We might want to consider renaming this configuration (either in this PR or a future one).

Reply:
I agree. Now I remove the dspace.ui configuration.